src/main/java/at/hannibal2/skyhanni/features/event/UniqueGiftingOpportunitiesFeatures.kt

```diff
@@ -71,16 +73,16 @@ object UniqueGiftingOpportunitiesFeatures {
         analyzeArmorStand(entity)
     }
 
-    @SubscribeEvent
-    fun onEntityJoinWorld(event: EntityEnterWorldEvent) {
+    @HandleEvent
+    fun onEntityJoinWorld(event: EntityEnterWorldEvent<Entity>) {
         playerColor(event)
         val entity = event.entity as? EntityArmorStand ?: return
         analyzeArmorStand(entity)
     }
 
-    private fun playerColor(event: EntityEnterWorldEvent) {
+    private fun playerColor(event: EntityEnterWorldEvent<Entity>) {
         if (event.entity is EntityOtherPlayerMP) {
-            val entity = event.entity as EntityOtherPlayerMP
+            val entity = event.entity
```

Comment:
the generic should be EntityOtherPlayerMP

Reply:
It cant be because this also calls analyzeArmorStand